### PR TITLE
Wrapping component with the GlobalStateProvider

### PR DESCRIPTION
**Pull request by mcastrigno**
For greater clarity, adding reminder to wrap the SomeComponent with the GlobalStateProvider.

### CODE REVIEW

File: docs/recipes/react.md [hunk 1]
```diff
@@ -157,6 +157,21 @@ export const SomeComponent = (props) => {
 
 Note that you don't need to call `useActor` for this, it's available right on the context.
 
+## Access to the Global State Provider
+In the above examples, in order for the component to have access to the global state it is required to wrap the component with the global state provider. 
+```js
+import SomeComponent from './someComponent'
+import GlobalStateProvider from './GobalStateProvider'
```
Comment (davidkpiano) [inline docs/recipes/react.md:164]:
```suggestion
import GlobalStateProvider from './GlobalStateProvider'
```

**Review by laurakalbag — COMMENTED** [review]
@davidkpiano I wonder if this information would be better included inside the `### Content Provider` section, where `GlobalStateProvider` is first mentioned?

**Comment by davidkpiano** [comment]
> @davidkpiano I wonder if this information would be better included inside the `### Content Provider` section, where `GlobalStateProvider` is first mentioned?

cc. @mattpocock

**Comment by mattpocock** [comment]
Have assigned to self, will rewrite this a little at some point to make it fit with the rest of the content.

**Comment by davidkpiano** [comment]
Superseded by `createActorContext` feature

**Pull request closed without merging**